assets/training/finetune_acft_hf_nlp/src/finetune/finetune.py

```diff
@@ -617,6 +657,47 @@ def main():
     parser = get_parser()
     args, _ = parser.parse_known_args()
 
+    # Copy the args generated in the preprocess step
+    args = copy_preprocess_args(args)
+
+    # Set logging parameters
+    set_logging_parameters(
+        task_type=args.task_name,
+        acft_custom_dimensions={
+            LoggingLiterals.PROJECT_NAME: PROJECT_NAME,
+            LoggingLiterals.PROJECT_VERSION_NUMBER: VERSION,
+            LoggingLiterals.COMPONENT_NAME: COMPONENT_NAME
+        },
+        azureml_pkg_denylist_logging_patterns=LOGS_TO_BE_FILTERED_IN_APPINSIGHTS,
+    )
+
+    # Updata lora target modules for falcon
+    update_lora_target_modules()
```

Comment:
Can we do this only if the model is Falcon?

Reply:
done